### PR TITLE
Jailbird Break fix

### LINKED ISSUE #1858: [BUG] Jailbird break method doesn't work
**Describe the bug**
No any errors, it just does nothing.
In master was this code and it works fine
```
        public void Break()
        {
            Base._broken = true;
            Base.SendRpc(JailbirdMessageType.Broken);
        }
```

**EXILED Version ("latest" is not a version):**
7.0.0-rc.5

### PR DESCRIPTION
**Pull request by Vladislav-CS**
#1858
Has to be tested

### CODE REVIEW

File: Exiled.API/Features/Items/Jailbird.cs [hunk 1]
```diff
@@ -110,7 +110,10 @@ public JailbirdWearState WearState
         /// <summary>
         /// Breaks the Jailbird.
         /// </summary>
-        public void Break() => WearState = JailbirdWearState.Broken;
+        public void Break()
+        {
+            Base.SendRpc(JailbirdMessageType.Broken);
```
Comment (NaoUnderscore) [inline Exiled.API/Features/Items/Jailbird.cs:115]: Keep the lambda: `public void Break() => Base.SendRpc(JailbirdMessageType.Broken);`

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Items/Jailbird.cs (hunk 1 above)
Comment (louis1706) [reply Exiled.API/Features/Items/Jailbird.cs:115]: actually i don't even see how the Item will be destroy it's just send Rpc to client

**Comment by Thundermaker300** [comment]
Shouldn't this also have `Base._broken = true;`? It only sends RPC, doesn't actually break it

**Comment by louis1706** [comment]
> Shouldn't this also have `Base._broken = true;`? It only sends RPC, doesn't actually break it

that what i said in nao Requested Change

**Review by louis1706 — APPROVED** [review]

**Review by louis1706 — COMMENTED** [review]
I would just like than you test this i am not sure if the Item will imidiately disapear when destroy

**Pull request merged**